src/dishka/integrations/starlette.py

```diff
@@ -29,12 +30,19 @@ def __init__(self, app: ASGIApp) -> None:
     async def __call__(
             self, scope: Scope, receive: Receive, send: Send,
     ) -> None:
-        if scope["type"] != "http":
+        if scope["type"] not in ("http", "websocket"):
             return await self.app(scope, receive, send)
 
-        request = Request(scope)
+        if scope["type"] == "http":
+            request = Request(scope)
+            context = {Request: request}
+
+        else:
+            request = WebSocket(scope, receive, send)
+            context = {WebSocket: request}
+
         async with request.app.state.dishka_container(
```

Comment:
IMHO, we should enter SESSION scope for websocket so each message will be REQUEST

Reply:
Hmm, I agree, it looks more correct this way